bundle/language_en.properties

```diff
@@ -665,24 +665,56 @@ angal.hospital.visitduration.txt
 angal.hospital.visitendhour.txt                                                                        = Visit end hour
 angal.hospital.visitstarthour.txt                                                                      = Visit start hour
 angal.lab.allnegative.txt                                                                              = All negative.
+angal.invetory.allmedicaladdedsuccessfully.msg                                                         = All medical are added successfully.
 angal.inventory.allproduct.radio                                                                       = All products
+angal.inventory.canceled                                                                               = Canceled
```

Comment:
Where is this used?

Reply:
It's used in the InventoryWardBrowser file on line 563